src/sass/shared/mixins.scss

```diff
@@ -80,6 +93,20 @@
   }
 }
 
+@mixin input($background-color: $color-medium-grey) {
+  height: $height-slack-form-input;
+  border-radius: 25px;
+  background-color: $background-color;
+  border: 2px solid $color-slack-form-input-border;
+  font-size: $font-size-body;
+  font-family: $font-default;
+  padding: 0 15px;
```

Comment:
seems related with one of:
- $margin-bottom-slack-form-input: 15px;
- $margin-bottom-slack-form-submit-button: 15px;
- $margin-bottom-slack-form-message: 15px;

from `variables.scss`